### ISSUE TITLE
Shortcodes being output in Open Graph descriptions

### ISSUE THREAD

**Issue by wpsmort** [semperfiwebdesign/all-in-one-seo-pack#2004]
Reported here - https://semperplugins.com/support/troubleshooting-all-in-one-seo-pack-pro/how-to-hide-short-codein-autogenerate-descriptions-when-i-share-it-on-facebook/

Since updating to the latest release (Free v2.9 / Pro v2.10), sharing to Facebook shows shortcodes.

**Comment by wpsmort** [comment]
Also reported here - https://wordpress.org/support/topic/facebook-description-post/

**Comment by wpsmort** [comment]
For the second report above (from wordpress.org), I got access to the user's site and verified that he has Elementor page builder which uses shortcodes.  He did not have Run Shortcodes in Autogenerated Descriptions enabled.  Enabling this fixed the issue.  Note that he did not have the Social Meta module enabled so it looks like Facebook was using the meta description when he shared posts on Facebook

I've determined that the code we removed in PR #1964 caused this.  

In v2.8 when Autogenerate Descriptions is enabled and Run Shortcodes is not, we did not output a meta description or OG description in the source code.  I verified on a test site.

In v2.9 with the same settings, we now output a meta description or OG description in the source code.  It's this change in behavior that is causing users problems if they don't have Run Shortcodes enabled.  

I've tested with PR #1998 and confirmed this fixes this problem.  This reverts back to no meta description or OG description in the source code unless Run Shortcodes is enabled.
